epistles-v1/original/09-epistles.adoc

```diff
@@ -919,17 +929,24 @@ and so hath unity, and that hath the kingdom.
 [.signed-section-signature]
 G+++.+++ F.
 
+[.postscript]
+====
+
 All Friends, ye must come into a temperance above all the world.
```

Comment:
I wasn't sure what to do with this. In the original, it has each of these starting on their own line, but they all looked funny offset.

Reply:
Yeah, i tried this literally about 8 different ways, and I I don't like any of them. haha!

I think though, my preference after spending too much time on it is, if we added an `[.offset]` above every line starting with `All Friends,...` (except for the first, which is not necessary, because it's in a postscript).